internal/services/mssql/mssql_virtual_machine_resource.go

```diff
@@ -868,11 +875,16 @@ func expandSqlVirtualMachineTempDbSettings(input []interface{}) *sqlvirtualmachi
 	if len(input) == 0 || input[0] == nil {
 		return nil
 	}
-	dataStorageSettings := input[0].(map[string]interface{})
+	tempDbSettings := input[0].(map[string]interface{})
 
 	return &sqlvirtualmachine.SQLTempDbSettings{
-		Luns:            expandSqlVirtualMachineStorageSettingsLuns(dataStorageSettings["luns"].([]interface{})),
-		DefaultFilePath: utils.String(dataStorageSettings["default_file_path"].(string)),
+		luns:              	expandSqlVirtualMachineStorageSettingsLuns(tempDbSettings["luns"].([]interface{})),
+		data_file_count:   	utils.Int32(int32(tempDbSettings.Get("data_file_count").(int))),
+		data_file_size_mb: 	utils.Int32(int32(tempDbSettings.Get("data_file_size_mb").(int))),
+		data_file_growth_in_mb: utils.Int32(int32(tempDbSettings.Get("data_file_growth_in_mb").(int))),
+		default_file_path: 	utils.String(tempDbSettings["default_file_path"].(string)),
+		log_file_size_mb: 	utils.Int32(int32(tempDbSettings.Get("log_file_size_mb").(int))),
+		log_file_growth_mb: 	utils.Int32(int32(tempDbSettings.Get("log_file_growth_mb").(int)))
```

Comment:
field name of [SQLTempDbSettings](https://github.com/Azure/azure-sdk-for-go/blob/f4fc5f70ccfd445da5c860fb0e24fd96afa38119/services/preview/sqlvirtualmachine/mgmt/2021-11-01-preview/sqlvirtualmachine/models.go#L1527) needs to be corrected, e.g. `data_file_count` ->`DataFileCount`

Reply:
Ok this should be resolved